<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSaleDialog</class>
 <widget class="QDialog" name="NewSaleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Sale</string>
  </property>
  <widget class="QListWidget" name="listAllItems">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>531</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtItemCode">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblItemCode">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Item Code:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnAddItem">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add
Item</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnRemoveItem">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Remove
Item</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnListAllItems">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>List
All
Items</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCloseSale">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close
Sale</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotal">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>480</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total: 
0.00</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCheckout">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Checkout</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtItemCode</tabstop>
  <tabstop>btnAddItem</tabstop>
  <tabstop>btnRemoveItem</tabstop>
  <tabstop>btnListAllItems</tabstop>
  <tabstop>btnCheckout</tabstop>
  <tabstop>btnCloseSale</tabstop>
  <tabstop>listAllItems</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
